carservice/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from django.core.validators import MaxValueValidator
+
+
+class Car(models.Model):
+    serial_number = models.CharField(max_length=17)
+    car_mileage = models.PositiveIntegerField(validators=[MaxValueValidator(300000)])
+    car_model = models.CharField(max_length=30)
+    car_year = models.DateField()
+
+    def __str__(self):
+        return f'Model: {self.car_model}, serial number:({self.serial_number})'
+
+
+class Offer(models.Model):
+    description = models.TextField()
+    price = models.FloatField(validators=[MaxValueValidator(100000.0)])
+
+    id_car = models.ForeignKey(Car, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return f'Price: ({self.price}), car:({self.id_car})'
+
+
+class User(models.Model):
+    name = models.CharField(max_length=30)
+    surname = models.CharField(max_length=30)
+    username = models.CharField(max_length=30)
+    password = models.CharField(max_length=30)
+
+    def __str__(self):
+        return f'Name: {self.name} {self.surname}'
+
+
+class Rent(models.Model):
+    status = models.BooleanField(default=True)
+    rent_duration = models.DurationField()
```

Comment:
nie wiem czy to nie będzie łatwiejsze jak się rozbije to pole na dwa datetime start i end